pyproject.toml

```diff
@@ -35,9 +35,10 @@ Pillow = "*"
 pycryptodome = "*"
 PyJWT = "*"
 python = ">=3.8,<4"
+python-levenshtein = "^0.21.1"
```

Comment:
This package was renamed to `levenshtein`. The old name works, but it causes some weirdness sometimes. I think switching to use `levenshtein` here would be a good idea, since we're adding it in now. 

See the note at: https://pypi.org/project/python-Levenshtein/
> :warning: The package was renamed to Levenshtein and can be found [here](https://github.com/maxbachmann/Levenshtein).

Reply:
Good catch. Updated.